### DISCUSSION TITLE
1-Click Installations

### DISCUSSION THREAD

**Discussion by andrewcouling** [timschofield/webERP#274] [Development Discussion & Specification]
I was shopping for a new NAS device recently, and noticed that QNAP have webERP available within their '[app store](https://www.qnap.com/en-uk/app-center?os=qts&version=4.5.1&kw=weberp)' as a 1-click installation, albeit an old v4.13.x version.

I investigated further and found that Softaculous are a popular provider of 1-click install apps to web hosting companies. They currently have 4.15.2 available in their platform, with a [demo site](https://www.softaculous.com/apps/erp/webERP) which seems to work well.

May I suggest we make a mental note to notify these companies (and others) once v5 has launched?

**Comment by dalers** [comment]
Hi @andrewcouling I wasn't aware of the QNAP app store. Does installing webERP from QNAP also include MariaDb or MySQL?

I stumbled across the Softaculus catalog a couple weeks ago, and it didn't seem a Softaculus webERP install included a database, but I could easily have been mistaken. What was your impression?

Does anyone (i.e. @timschofield ;-) ) know who has the Softaculus credentials for when (and if) we want to update Softaculus to webERP v5, or how webERP got to be in the QNAP app store?

**Reply by andrewcouling** [reply]
> Does installing webERP from QNAP also include MariaDb or MySQL?

I do not know. I went for a Synology NAS in the end, rather than QNAP, so I'm not able to look into it.

**Comment by timschofield** [comment]
I did some work with the people from QNAP a few years ago when they were
first including it. I'm pretty sure back then they were using mysql, but it
might have changed. However if they are still using 4.13 then that is still
the version they were back then, so probably not much has changed regarding
their DBMS/server etc.

Thanks
Tim

On Thu, 5 Dec 2024 at 00:35, Andrew Couling ***@***.***>
wrote:

> Does installing webERP from QNAP also include MariaDb or MySQL?
>
> I do not know. I went for a Synology NAS in the end, rather than QNAP, so
> I'm not able to look into it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/274#discussioncomment-11467365>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LH2ESUO6OD4JXJVATL2D6NVFAVCNFSM6AAAAABTBIBHM6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBWG4ZTMNI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by timschofield** [comment]
I know that FrontAccounting (a very early fork from webERP) used to have
packages in the Ubuntu repositories which did a one click install -
obviously only for Linux. A quick look shows that they appear to have been
dropped. They also had a deal with some hosting companies to offer their
clients access to it, but a very quick scan doesn't seem to show any. Maybe
once we have a stable package we could try and fill this void?

Tim

On Sun, 8 Dec 2024 at 08:18, Tim Schofield ***@***.***>
wrote:

> I did some work with the people from QNAP a few years ago when they were
> first including it. I'm pretty sure back then they were using mysql, but it
> might have changed. However if they are still using 4.13 then that is still
> the version they were back then, so probably not much has changed regarding
> their DBMS/server etc.
>
> Thanks
> Tim
>
> On Thu, 5 Dec 2024 at 00:35, Andrew Couling ***@***.***>
> wrote:
>
>> Does installing webERP from QNAP also include MariaDb or MySQL?
>>
>> I do not know. I went for a Synology NAS in the end, rather than QNAP, so
>> I'm not able to look into it.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/timschofield/webERP/discussions/274#discussioncomment-11467365>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AAAL6LH2ESUO6OD4JXJVATL2D6NVFAVCNFSM6AAAAABTBIBHM6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBWG4ZTMNI>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
> www.weberp.org <http://www.weberpafrica.com>
> @TimSchofield2
> Blog: https://kwamoja.home.blog/
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/
